frontend/apps/marketing/src/components/actionBlocks/fullWidthActionBlock/FullWidthActionBlock.tsx

```diff
@@ -0,0 +1,71 @@
+import {BaseEntry, EntryFields} from 'contentful';
+
+import DSCOFullWidthActionBlock, {
+  ActionBlockProps,
+} from '@code-dot-org/component-library/actionBlock/fullWidthActionBlock';
+
+export type FullWidthActionBlockContentfulProps = ActionBlockProps & {
+  image: BaseEntry & {
+    fields: {
+      description?: EntryFields.Text;
+      title?: EntryFields.Text;
+      file: {url: EntryFields.Text};
+    };
+  };
+  overline: EntryFields.Text;
+  title: EntryFields.Text;
+  description: EntryFields.Text;
+  primaryButton: BaseEntry & {
+    fields: {
+      label: EntryFields.Text;
+      primaryTarget: EntryFields.Text;
+      ariaLabel: EntryFields.Text;
+    };
+  };
+  secondaryButton: BaseEntry & {
+    fields: {
+      label: EntryFields.Text;
+      primaryTarget: EntryFields.Text;
+      ariaLabel: EntryFields.Text;
+    };
+  };
+  background: EntryFields.Text;
+};
```

Comment:
Just letting you know that once https://github.com/code-dot-org/code-dot-org/pull/64911 will be merged, I plan to use `ImageAssetEntry` and `LinkEntry` types for links and image entries instead of declaring them manually

Might also update that to use EntryFields types instead of native ts types where applicable

Reply:
Nice!! I think it's worth updating those to use the `EntryFields` types (tbh might be worth updating every component to use that, but we can revisit later 😅)